src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -456,13 +456,21 @@ void LottieParser::parsePropertyInternal(T& prop)
 }
 
 
-template<typename T>
-void LottieParser::parseProperty(T& prop)
+template<LottieProperty::Type type, typename T>
+void LottieParser::parseProperty(T& prop, LottieObject* obj)
 {
     enterObject();
     while (auto key = nextObjectKey()) {
         if (!strcmp(key, "k")) parsePropertyInternal(prop);
-        else skip(key);
+        else if (obj && !strcmp(key, "sid")) {
+            auto slot = new LottieSlot;
+            if (!slot) return;
+
+            slot->sid = getStringCopy();
+            obj->sid = slot->sid;
+            slot->type = type;
+            comp->slots.push(*slot);
```

Comment:
Please check the next, comp.slots doesn't have elements with the memory pointer anymore.
```
comp->slots.push({getStringCopy(), type});
obj->sid = comp->slots.last().sid;
```